civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -4,6 +4,15 @@
  */
 
 (function ($) {
+  // Add civicrm-page class to body of onboarding form page
+  $(document).ready(function () {
+    var currentPath = $(location).attr('pathname');
```

Comment:
Please move that to a separate function. The comment you wrote will go to the function description:

```js
(function ($) {
  $(document).ready(function () {
    someFunc();
  });
  ...

/**
 * Description
 */
function someFunc () {
 ...
}
```

Reply:
No need of this code now, as there is no need to add `page-civicrm` class to body of onboarding form page .Thanks!  though.